### PR TITLE
remove some expect

### PR DESCRIPTION
**Pull request by librelois**
Remove some expect in import queue

### CODE REVIEW

**Review by JoshOrndorff — APPROVED** [review]
I'm curious. Is this just general code quality improvement, or did you encounter some panics or errors?

**Comment by notlesh** [comment]
We should be careful to make sure this doesn't change behavior when it's important to produce an invalid block when we hit certain cases.

panic: definitely produces an invalid block
return Err: up to caller

**Review by girazoki — APPROVED** [review]

File: nimbus-consensus/src/lib.rs [hunk 1]
```diff
@@ -222,14 +222,12 @@ where
...
-			&*client.runtime_api(),
-			&at,
-			NimbusId::from_slice(&type_public_pair.1).expect("Provided keys should be valid"),
-			slot_number,
-			parent,
-		)
-		.expect("NimbusAPI should not return error")
+		if let Ok(nimbus_id) = NimbusId::from_slice(&type_public_pair.1) {
+			NimbusApi::can_author(&*client.runtime_api(), &at, nimbus_id, slot_number, parent)
+				.unwrap_or_default()
```
Comment (notlesh) [inline nimbus-consensus/src/lib.rs:227]: `unwrap_or(false)` might be more clear here (and less relient on defined types), although it's technically the same thing

**Review by notlesh — APPROVED** [review]

File: nimbus-consensus/src/lib.rs (hunk 1 above)
Comment (librelois) [reply nimbus-consensus/src/lib.rs:227]: bool default is false since the begin of the rust language and that will never change

Comment (librelois) [reply nimbus-consensus/src/lib.rs:227]: What I mean is that relying on rust primitives types like `bool` is safe, theses types will not change

**Pull request merged**